jarbas/settings.py

```diff
@@ -27,8 +27,8 @@
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = config('SECRET_KEY')
 
-# SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = config('DEBUG', default=True, cast=bool)
```

Comment:
The [`DEBUG` setting](https://docs.djangoproject.com/en/2.1/ref/settings/#std:setting-DEBUG) is a convention in Django and many third-parties depends on it (WhiteNoise, Debug Toolbar etc). I would not recommend to delete, but we could safely remove it from the environment and use the `ENVIRONMENT` to deduce if debug mode is on or off:

```python
DEBUG = config('ENVIRONMENT', cast=lambda x: x == 'development')
```

What do you think?

Reply:
Since I've been using the `ENVIRONMENT` for using or not the WhiteNoise middleware, I believe this will address your concerns in the same way.

```diff
diff --git a/jarbas/settings.py b/jarbas/settings.py
index 931e9949..802ec795 100644
--- a/jarbas/settings.py
+++ b/jarbas/settings.py
@@ -29,6 +29,7 @@ SECRET_KEY = config('SECRET_KEY')

 ENVIRONMENT = config('ENVIRONMENT', default='development')
 LOG_LEVEL = config('LOG_LEVEL', default='debug')
+DEBUG = ENVIRONMENT != 'production'

 ALLOWED_HOSTS = config('ALLOWED_HOSTS', default='*', cast=Csv())
 INTERNAL_IPS = ('127.0.0.1',)
```